app/src/main/java/com/codesoom/assignment/models/Task.java

```diff
@@ -1,9 +1,13 @@
 package com.codesoom.assignment.models;
 
+/**
+ * 할 일
+ */
```

Comment:
JavaDoc으로 빌드하면 "공식" 문서가 되기 때문에 이런 점에 신경쓸 수 있어야 합니다.
```suggestion
/**
 * 할 일.
 */
```

Reply:
공식문서.. 뭔가 부담감이 확 밀려 오는군요..
다음부터 세심하게 체크하겠습니다!